package.json

```diff
@@ -193,7 +193,7 @@
 		"react-test-renderer": "16.13.1",
 		"rimraf": "3.0.2",
 		"rtlcss": "2.6.2",
-		"sass": "1.26.11",
+		"sass": "1.26.12",
```

Comment:
Unintentional change here? Along with the package lock file.

Reply:
Oh I see why you made this update now.

Reply:
I included this to force the cache to clear with a changed lock file. It was a super minor update to the dependency and should be totally fine to include. I can revert that if we want to avoid this change. The benefit of this change just won't be realized until after the next time the lock file changes.

Reply:
Yep! We simultaneously commented in case someone finds this later 😂

Reply:
I assumed that it was added to measure the impact with https://github.com/WordPress/gutenberg/pull/30510/commits/db2cbb5c00f5a41df5b70a7214ef9619c4990261 😄